common4j/src/main/com/microsoft/identity/common/java/cache/MsalCppOAuth2TokenCache.java

```diff
@@ -139,13 +138,6 @@ public synchronized void saveCredentials(@Nullable final AccountRecord accountRe
             }
         }
 
-        if (accountRecord != null && refreshTokenRecord != null) {
-            // MSAL C++ writes credentials first and then the account.
```

Comment:
Not sure I understand this comment.  Did we wask Brian why this was done before removing it?

Reply:
As I recall, this was written to ensure that a single RT is present in the cache for a user at any given time (we're removing any existing RT before writing a new one). By removing this, the assertion appears to be either that this will always be the case anyway (absent some kind of enforcement) or that this doesn't matter.

If the newly written RT always has the same cache key as the old one, this can be removed without changing the effect of the method.